### ISSUE TITLE
Port in the API from the locp/cassandra module

### ISSUE THREAD

**Issue by dallinb** [locp/opscenter#1]
This will be an initial release of the module with an identical API to that of the locp/cassandra-1.X interface.

**Comment by dallinb** [comment]
```
  1) opscenter class Test installation. Should run without error
     Failure/Error:
       apply_manifest(install_pp,
                      catch_failures: true)
     Beaker::Host::CommandFailure:
       Host 'debian7' exited with 1 running:
        puppet apply --verbose --detailed-exitcodes /tmp/apply_manifest.pp.QEgfvz
       Last 10 lines of output were:
           (at /usr/lib/ruby/vendor_ruby/puppet/settings.rb:1139:in `issue_deprecation_warning')
        Info: Loading facts
        Info: Loading facts
        Info: Loading facts
        Notice: Compiled catalog for debian7.locp.co.uk in environment production in 0.69 seconds
        Info: Applying configuration version '1464679937'
        Info: Creating state file /var/lib/puppet/state/state.yaml
        Error: Failed to apply catalog: Found 1 dependency cycle:
        (File[/etc/opscenter/clusters/Cluster1.conf] => Ini_setting[Cluster1:cassandra_seed_hosts] => Service[opscenterd] => Class[Opscenter] => Opscenter::Cluster_name[Cluster1] => File[/etc/opscenter/clusters/Cluster1.conf])
        Try the '--graph' option and opening the resulting '.dot' file in OmniGraffle or GraphViz

     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/host.rb:318:in `exec'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/dsl/helpers/host_helpers.rb:73:in `block in on'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/shared/host_manager.rb:115:in `run_block_on'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/dsl/patterns.rb:35:in `block_on'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/dsl/helpers/host_helpers.rb:63:in `on'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/dsl/helpers/puppet_helpers.rb:469:in `block in apply_manifest_on'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/shared/host_manager.rb:115:in `run_block_on'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/dsl/patterns.rb:35:in `block_on'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/dsl/helpers/puppet_helpers.rb:398:in `apply_manifest_on'
     # /home/ben/.rvm/gems/ruby-2.1.6/gems/beaker-2.33.0/lib/beaker/dsl/helpers/puppet_helpers.rb:476:in `apply_manifest'
     # ./spec/acceptance/opscenter_spec.rb:101:in `block (3 levels) in <top (required)>'

Finished in 45.52 seconds (files took 2 minutes 32.3 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/acceptance/opscenter_spec.rb:100 # opscenter class Test installation. Should run without error

/home/ben/.rvm/rubies/ruby-2.1.6/bin/ruby -I/home/ben/.rvm/gems/ruby-2.1.6/gems/rspec-core-3.4.4/lib:/home/ben/.rvm/gems/ruby-2.1.6/gems/rspec-support-3.4.1/lib /home/ben/.rvm/gems/ruby-2.1.6/gems/rspec-core-3.4.4/exe/rspec spec/acceptance --color failed
```

**Comment by dallinb** [comment]
Released to Puppet Forge as v0.1.0 so far.
